### PR TITLE
{bio}[GCCcore/9.3.0] SeqAn v2.4.0

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)
updated from #10065

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node374.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/70562f251798453ea685198c34260392 for a full test report.

File: easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+]
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/seqcons2', 'share/man/man1/sak.1'],
+    'dirs': []
```
Comment (boegel) [inline easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-9.3.0.eb]: @smoors @JackPerdue Existing `SeqAn` easyconfigs also copy the `include` dir + `'util/cmake/seqan-config.cmake'`, so should be done here too..

Comment (JackPerdue) [reply easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-9.3.0.eb]: I do not know why I did not see pre-existing SeqAn build.   Apologies to @pescobar for claiming authorship.   I will resubmit with those postinstallcmds and restore authorship to Pablo.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2409.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0b20d9af034abfaf4cee31bd4a5e30e6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-4 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/969560c889836746a20da0b712fe495b for a full test report.

File: easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-9.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-9.3.0.eb]: @JackPerdue Would be easier if @smoors can fix this PR...

Comment (JackPerdue) [reply easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-9.3.0.eb]: OK... all I ask is that my "Author" line be removed and Pablo's restored.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node352.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/a9aa98b5d66e5a77aa25e8e022842dbb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2491.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7c5127383e8e635b364c3c6416fdc620 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3114.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/8123e586a5662da46a0f1661b40c1851 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**